src/backend/dx11/src/data.rs

```diff
@@ -205,19 +206,21 @@ pub fn map_bind(bind: Bind) -> D3D11_BIND_FLAG {
 }
 
 
-pub fn map_access(access: MapAccess) -> D3D11_CPU_ACCESS_FLAG {
+pub fn map_access(access: mapping::Access) -> D3D11_CPU_ACCESS_FLAG {
     match access {
-        MapAccess::Readable => D3D11_CPU_ACCESS_READ,
-        MapAccess::Writable => D3D11_CPU_ACCESS_WRITE,
-        MapAccess::RW => D3D11_CPU_ACCESS_READ | D3D11_CPU_ACCESS_WRITE,
+        mapping::READABLE => D3D11_CPU_ACCESS_READ,
```

Comment:
Ah, I see that your new `Access` is a bit set, nevermind!
Still, I think it would be clearer here to check for each bit separately and append the corresponding `D3D11_CPU_ACCESS_` variant. For example, the current function would crash on the zero flag.